example/lib/main.dart

```diff
@@ -9,13 +13,11 @@ void main() {
 class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return MaterialApp(
+    return CupertinoApp(
```

Comment:
Why `CupertinoApp`? Does it not work with `MaterialApp`?

Reply:
I think those were some leftovers from testing, changed it back though.